spec/integration/client_auto_encryption_options_spec.rb

```diff
@@ -129,32 +136,94 @@
 
       expect(client.mongocryptd_client.options[:monitoring_io]).to be false
     end
-  end
 
-  context 'with default values' do
-    let(:auto_encryption_options) do
-      {
-        key_vault_namespace: key_vault_namespace,
-        kms_providers: kms_providers,
-        schema_map: schema_map,
-      }
-    end
-
-    it 'sets key_vault_client as self' do
-      expect(client.encryption_options[:key_vault_client]).to eq(client)
+    context 'with default extra options' do
+      let(:auto_encryption_options) do
+        {
+          key_vault_namespace: key_vault_namespace,
+          kms_providers: kms_providers,
+          schema_map: schema_map,
+        }
+      end
+
+      it 'sets key_vault_client as self' do
+        expect(client.encryption_options[:key_vault_client]).to eq(client)
+      end
+
+      it 'sets bypass_auto_encryption to false' do
+        expect(client.encryption_options[:bypass_auto_encryption]).to be false
+      end
+
+      it 'sets extra options to defaults' do
+        client_options = client.encryption_options
+
+        expect(client_options[:mongocryptd_uri]).to eq('mongodb://localhost:27020')
+        expect(client_options[:mongocryptd_bypass_spawn]).to be false
+        expect(client_options[:mongocryptd_spawn_path]).to eq('mongocryptd')
+        expect(client_options[:mongocryptd_spawn_args]).to eq(['--idleShutdownTimeoutSecs=60'])
+      end
     end
+  end
 
-    it 'sets bypass_auto_encryption to false' do
-      expect(client.encryption_options[:bypass_auto_encryption]).to be false
+  describe 'Spawning mongocryptd' do
+    require_enterprise
+
+    let(:client) do
```

Comment:
@p-mongo It seems like this client is not getting cleaned up by `close_local_clients` and it is causing errors -- am I doing something wrong here when creating the new local client?